README.rst

```diff
@@ -17,15 +17,21 @@ Usage
     $ rio mbtiles --help
     Usage: rio mbtiles [OPTIONS] INPUT [OUTPUT]
 
-      Export a dataset to MBTiles (version 1.1) in a SQLite file.
+      Export a dataset to MBTiles (version 1.3) in a SQLite file.
 
       The input dataset may have any coordinate reference system. It must have
       at least three bands, which will be become the red, blue, and green bands
       of the output image tiles.
 
       An optional fourth alpha band may be copied to the output tiles by using
-      the --rgba option in combination with the PNG format. This option requires
-      that the input dataset has at least 4 bands.
+      the --rgba option in combination with the PNG or WEBP formats. This option
+      requires that the input dataset has at least 4 bands.
+
+      The default quality for JPEG and WEBP output (possible range: 10-100) is
+      75. This value can be changed with the use of the QUALITY creation option,
+      e.g. `--co QUALITY=90`.  The default zlib compression level for PNG output
+      (possible range: 1-9) is 6. This value can be changed like `--co
+      ZLEVEL=8`.  Lossless WEBP can be chosen with `--co LOSSLESS=TRUE`.
```

Comment:
The important creation options are documented.

Reply:
Different libraries can measure compression quality in different ways, so there’s a potential gotcha where a user tests with a different webp library, decides that (say) 60% quality is good, and applies that number here, where it might be higher or lower than they intended.

However, I think that (a) that possibility is fairly unlikely, and (b) a reasonable person for whom slight quality differences matter will presumably do a web search for `gdal webp driver` in the first place. So I’m going to say 👍 but note my slight hesitation.

Reply:
Yes. I checked one other software package, Pillow, and we're consistent (though Pillow has "zlib_compression" instead of "zlevel"). What we have in our favor is that higher level software using libpng, libwebp, libjpeg will likely default to using the parameters of the low level libraries.